packages/framework/src/Model/Complaint/Status/ComplaintStatus.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Complaint\Status;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Prezent\Doctrine\Translatable\Annotation as Prezent;
+use Shopsys\FrameworkBundle\Model\Complaint\Status\Exception\ComplaintStatusDeletionForbiddenException;
+use Shopsys\FrameworkBundle\Model\Localization\AbstractTranslatableEntity;
+
+/**
+ * @ORM\Table(name="complaint_statuses")
+ * @ORM\Entity
+ * @method \Shopsys\FrameworkBundle\Model\Complaint\Status\ComplaintStatusTranslation translation(?string $locale = null)
+ */
+class ComplaintStatus extends AbstractTranslatableEntity
+{
+    /**
+     * @var int
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     */
+    protected $id;
+
+    /**
+     * @var \Doctrine\Common\Collections\Collection<int, \Shopsys\FrameworkBundle\Model\Complaint\Status\ComplaintStatusTranslation>
+     * @Prezent\Translations(targetEntity="Shopsys\FrameworkBundle\Model\Complaint\Status\ComplaintStatusTranslation")
+     */
+    protected $translations;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string", length=25)
+     */
+    protected $status;
```

Comment:
is not this rather `type` or `statusType`? I mean, if it works the same as order statuses, would it not be weird to have multiple statuses with the status "in progress"? Let's talk about it, please

Reply:
...maybe even the enum should be rather `ComplaintStatusTypeEnum` :thinking:

Reply:
renamed to statusType in 36a5d87371 (and fixed a bug 😇 )